src/app/dataset/[slug]/page.tsx

```diff
@@ -79,26 +102,25 @@ export default function Page({
                         <BreadcrumbItem>
                             <BreadcrumbLink
                                 className="text-xl font-bold"
-                                href="/dataset/Parkeringsplatser"
+                                href={`/dataset/${parsedPageData && datasetWithUnderscore(parsedPageData.records.title)}`}
```

Comment:
same as above, links should be slugs. so lowercase with dashes and not underscore